### DISCUSSION TITLE
[Feature Request] Add "view" command to the Handling (exclude, DELETE, REPORT, ...) menu

### DISCUSSION THREAD

**Discussion by akhcade** [ThioJoe/YT-Spammer-Purge#277] [Ideas]
Suggestion to add the following command:

\> To **view all comments from one of the authors**: Type 'view' followed by a number **in the sample list** next to the author you want to view.
\> Example: view 5

Ideally, this would list the username on one line, followed by a list of comment texts (one comment per line?). This is intended for authors where the sample picked is ambiguous (might be spam, might not).

**Comment by ThioJoe** [comment]
I'll possibly add something like this, but in the mean time you can just open the log file to look at the full comments
